apps/src/lab2/views/components/Instructions.tsx

```diff
@@ -217,6 +216,7 @@ const InstructionsPanel: React.FunctionComponent<InstructionsPanelProps> = ({
   return (
     <div
       id="instructions"
+      data-theme="Light"
```

Comment:
Same here, is this needed?

Reply:
Since these instructions still have a white background, this helps ensure that we use light mode buttons rather than dark mode ones.  For example, the primary purple button is actually white in dark mode, which doesn't work against the white background.